### PR TITLE
initial version of teddies_shop based on wx_pay

### PR DESCRIPTION
**Pull request by Eric-Guo**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,38 @@
+source 'https://gems.ruby-china.org'
+ruby '~> 2.4.0'
+
+gem 'rails', '~> 5.1.0.beta1'
```
Comment (ssaunier) [inline Gemfile]: Is there any reason you need to use the beta version?

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+gem 'uglifier'
+gem 'bootstrap-sass'
+gem 'font-awesome-sass'
+gem 'simple_form'
+gem 'autoprefixer-rails'
+
+gem 'money-rails'
+
+# $ bundle config --local local.wx_pay /Users/guochunzhong/git/oss/wx_pay
+gem 'wx_pay', git: 'git@github.com:Eric-Guo/wx_pay.git', branch: :master
```
Comment (ssaunier) [inline Gemfile]: Why do you need a specific version of the gem?

File: .ruby-version [hunk 3]
```diff
@@ -0,0 +1 @@
+2.4.0
```
Comment (ssaunier) [inline .ruby-version]: We are still using `2.3.3` at Le Wagon.

File: app/assets/config/manifest.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+//= link_tree ../images
```
Comment (ssaunier) [inline app/assets/config/manifest.js:1]: I think this is something brought by Rails 5.1

Again, we'll wait the stable version to be released before teaching it.

**Review by ssaunier — CHANGES_REQUESTED** [review]
Overall, LGTM+ - Have some questions on gem versions

File: app/assets/config/manifest.js (hunk 4 above)
Comment (Eric-Guo) [reply app/assets/config/manifest.js:1]: Now downgrade to Rails 5.0.2 and ruby version is also not binding to 2.4, I do the pay testing and all works as expected.

For manifest.js, it still can be exist even in 5.0.2, so I still prefer keep it to make further upgrade easier.

**Comment by ssaunier** [comment]
👍

**Pull request merged**